### PR TITLE
[Blog] Spotlight for k8s upstream training in Japan

### PR DESCRIPTION
**Pull request by Okabe-Junya**
xref. https://github.com/kubernetes/website/pull/48039
see https://github.com/kubernetes/website/pull/48039#pullrequestreview-2380578582

### Description

We are the organizing team for k8s upstream training in Japan. see https://github.com/kubernetes-sigs/contributor-playground/tree/master/japan for more details
We would like to feature our efforts in k/website blog!

Deploy preview: https://deploy-preview-541--kubernetes-contributor.netlify.app/blog/2024/10/28/k8s-upstream-training-japan-spotlight/

**Important Date**

- We plan to move this PR out of draft status by ~~early October~~
- We would like to publish this blog post within ~~October~~ early November (November 10?)

Let me know if you’d like any adjustments!

### Issue

Closes: #

### CODE REVIEW

**Comment by sftim** [comment]
/hold

pending an agreed publication date being set

OK to unhold if the publication date is agreed and in the future

File: content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md
Comment (fsmunoz) [inline content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md:11]:
```suggestion
## About our team
```

File: content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md
Comment (fsmunoz) [inline content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md:20]:
```suggestion
## What is Kubernetes upstream training in Japan?
```

File: content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md
Comment (fsmunoz) [inline content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md:56]:
```suggestion
## Interview with participants
```

File: content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md
Comment (fsmunoz) [inline content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md:166]:
```suggestion
## Future of Kubernetes upstream training
```

**Review by fsmunoz — CHANGES_REQUESTED** [review]
すばらしい! This are just some nits to change the heading style to sentence case as per the style guide - the rest is a great read and looks ready to go.

**Comment by fsmunoz** [comment]
This changes can be made in a different PR though.

/lgtm

**Comment by sftim** [comment]
> [IMPORTANT NOTE] When you review, please comment on the https://github.com/kubernetes/website/pull/48039 to avoid discussing across multiple PRs.

Actually we prefer to have the comments on the PR that adds the primary / original copy. So, _this_ PR.

File: content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md
Comment (fsmunoz) [inline content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md:5]:
```suggestion
date: 2024-10-28
```

**Comment by fsmunoz** [comment]
Suggesting 28th October as publication date, right after the weekend. There's nothing against publishing it during the weekend, but it's not generally done due to traffic being less outside of the work week.

**Comment by Okabe-Junya** [comment]
Thank you all for the great reviews and supports!!

Based on @sandeepkanabar 's comments on k/website and @fsmunoz ’s feedback, I’ve made some changes.

Do we need a bit more consensus regarding the publication date? 
This PR is somewhat time-sensitive, but the event is still over a month away. However, if it could be published by 10/28, that would be great. 

I understand that there are PRs in the review queue and that reviewer resources are limited, thank you

/cc fsmunoz

**Comment by Okabe-Junya** [comment]
/assign @sftim @natalisucks 
for approval

File: content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md
Comment (sftim) [inline content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md:8]: I think this will make more sense
```suggestion
  [Junya Okabe](https://github.com/Okabe-Junya) (University of Tsukuba) / 
  Organizing team of Kubernetes Upstream Training in Japan
```

**Comment by sftim** [comment]
The author information looks a little odd; the article is attributed to:
[Junya Okabe](https://github.com/Okabe-Junya) (University of Tsukuba) Organizing team of Kubernetes Upstream Training in Japan

which could mean that Junya Okabe _is _ the organizing team, or is part of the team, or that the article has two coauthors: Junya Okabe and the organizing team.

It would be good to clarify which was meant.

Anyway:
/approve

I will unhold shortly. Please bear with me.

File: content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md
Comment (sftim) [inline content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md:11]: (nit)
We avoid starting articles with a heading. Not a rule we enforce but still preferred.

**Comment by sftim** [comment]
/hold cancel

The mirror article has LGTM and approval, too.

**Comment by Okabe-Junya** [comment]
Thanks! I'll fix some nits later :)

**Pull request merged**
